mautic/mautic
# Hubspot-bug-fixes

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
with changes to the config setting and direction being added, hubspot was not pushing/fetching leads correctly.
[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. try to fetch contacts and companies from hubspot, contacts pulled nothing, companies errored
2. try pushing through integration and it was probably not pushing correctly

#### Steps to test this PR:
1. Apply this PR try the fetchleads command for contacts and companies
`php app/console mautic:integration:fetchleads  --integration=Hubspot --time-interval='29 days' -e dev `
2. try pushing a contact to hubspot through a campaign and a form action, everything should work as desired
3. direction flow was added

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
